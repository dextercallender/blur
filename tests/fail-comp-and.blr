void main() {
	int a = 1;
	char c = 'c';
	if(a==c and a < 4) {
		println(a);
	}
}
