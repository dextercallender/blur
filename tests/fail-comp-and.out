Fatal error: exception Failure("invalid operands")
